### ISSUE TITLE
not able to use the alt feature

### ISSUE THREAD

**Issue by michaelhonein** [vinceliuice/WhiteSur-firefox-theme#35]
i succesfully installed the theme onto firefox but i cant figure out how to use the monterey theme on it. Anybody can help?

**Comment by AdamXweb** [comment]
What was your install method?
Did you manually copy files across?

**Comment by michaelhonein** [comment]
yes i manually moved the files from my downloads folder to my profile folder

**Comment by drkhero** [comment]
hey @AdamXweb , I sent you an email to **contact(at) privaceykey(dot)org**.

Please get back to me on it. I'm incredibly impressed by your work. Thank you!

**Comment by vinceliuice** [comment]
Run `./tweaks.sh -f alt` to install firefox alt theme @michaelhonein
